### ISSUE TITLE
mime type

### ISSUE THREAD

**Issue by Lesp1ker** [CulturalMe/meteor-slingshot#137]
1. if to load files except the image , there is a mistake:
"Cannot pre-load anything other than images" upload.js:275.
2. and "allowedFileTypes" not supported mime-type "text/plain"?
sorry for bad English

**Comment by gsuess** [comment]
Yes, you cannot pre-load `"text/plain"`. Only images can be pre-loaded. Do not request the image to be preloaded when calling `.url()` (Do not pass true as an argument).
